### DISCUSSION TITLE
【2025年11月】灵感漫溢的甜蜜创新 凹 正式pv

### DISCUSSION THREAD

**Discussion by Hinata-Mayasa** [share121/inter-knot#4527] [其他]
【重磅消息速递！！！】
Madosoft在今天正式宣布制作《常规脱离Creative·凹作》并且预计2026年年底中发售，
2024年9月7日，Madosoft的Facebook等多个平台的账号宣布制作常规脱离的凹作，根据帖子所讲：“根据在《常规脱离》中本作和凸作的成功，我们深刻意识到了角色性格塑造、懂得玩家所想和的满足玩家愿望重要性，特此献上常规脱离的第二款续作（也就是俗称的凹作）。”
这款作品被Madosoft视为夺得2027年GAL大赏的希望大作（或许是2021年被《爱上火车》超了不服气），同时也将是窗社有史以来投入最大、开发时间最长、制作人员最多、美术最精良已经故事最感人的作品，在其中不仅有在本作和凹作出现的主角线，更有莉莉丝、里姐等更多角色线路的加入以回应玩家们的对于角色的喜爱和支持。Madosoft创始人推特账号也在随后发帖，称其：“如果能够在此作取得巨大成功，那么常规脱离凹作将会是日本最为精良的gal。”
国内知名gal论坛Hikarinagi站长Yuki在消息放出后的10分钟发帖表示对于此作品十分期待，并且在帖子发出后短短1小时内就斩获30000+的阅读量，成为Hikarinagi阅读量增长最快的帖子。
详情请看B站搬运视频：
https://www.bilibili.com/video/BV1uT4y1P7CX/?spm_id_from=333.337.search-card.all.click&vd_source=ed6c1b36df7e94d0ad8e5493a45fc6d8
![8B87E32E9D72286322B8E3D87A30858F](https://github.com/user-attachments/assets/d74189bf-df61-41a3-91c1-6b7003a64a81)
![凹](https://github.com/user-attachments/assets/81930ecd-eea2-44aa-8f31-3845d309b640)

**Comment by LMA936** [comment]
被骗了（我自愿的）
